internal/bitwarden/bw/models.go

```diff
@@ -28,8 +28,9 @@ const (
 type ObjectType string
 
 const (
-	ObjectTypeItem   ObjectType = "item"
-	ObjectTypeFolder ObjectType = "folder"
+	ObjectTypeItem           ObjectType = "item"
+	ObjectTypeItemAttachment ObjectType = "attachment"
```

Comment:
This was actually meant for the values of the `object` field of the JSON representation of some objects. Since it doesn't exist for attachment, I would put the constant elsewhere.